src/buildtools/fsyacc/fsyacc.fs

```diff
@@ -12,7 +12,7 @@ open FsLexYacc.FsYacc
 open FsLexYacc.FsYacc.AST
 
 //------------------------------------------------------------------
-// This code is duplicated from FSharp.Compiler.UnicodeLexing
+// This code is duplicated from Microsoft.FSharp.Compiler.UnicodeLexing
```

Comment:
missed a namespace change here

Reply:
This is just copying over the canonical source code from FsLexYacc, we'd need to make the change there